pkg/webhooks/registration.go

```diff
@@ -17,6 +17,9 @@ import (
 	rest "k8s.io/client-go/rest"
 )
 
+// defaultWebhookTimeout = 2s
+var defaultWebhookTimeout = int32(2)
```

Comment:
can this be a configurable field ? as it might need to be fine tuned to each system? 
Command line arg or config map ?

Reply:
Do you see any type of cluster needs a different configuration?
For now maybe it should be consistent across clusters.

Reply:
the timeouts for client, is defined in client config and same for is applicable for kubectl.. so the question is if they are configurable would it be helpful to adjust based on the timeouts defined there? we can default it now, but this is something that should be allowed to be configured

Reply:
This is the timeout when apiserver calls webhook. Most of the case, the user won't know how this timeout would impact their cluster. It might be dangerous to expose this configuration to user, since it would bring unknown behavior to the cluster/kyverno. 
Maybe we can expose this for debugging/developing purpose.

Reply:
True, this configuration is not meant for user.
But the cluster admin who sets up kyverno for policy management